packages/angular_devkit/build_facade/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@angular-devkit/build-facade",
```

Comment:
Move the files and rename the package to `architect`, like we do with `schematics`.